apps/events.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: docs
+key: events
+title: Events
+---
+
+Events can be triggered when a customer executes a specific action in an ePages online shop.
+This can be e.g. a customer opening a new page of a shop, or adding a product to the cart.
+If you have implemented an event in your application, you will be informed about related actions and can react accordingly.
+Furthermore, your app will receive information related to the event, such as the current page a user is on, or which product was added to the cart.
```

Comment:
Technically this is wrong. The will not receive which product was added to the cart. Only the latest state of the cart is shared.